File: latex/analysis.py
import matplotlib.pyplot as plt
import pandas as pd


normal = pd.read_csv('normal.csv', header=None)

normal.hist()
plt.savefig('normal_hist.pdf')

q = normal.quantile([0, 0.25, 0.5, 0.75, 1])
q.columns = ['quantiles']
q.to_latex('quantiles.tex')

File: latex/R/analysis.R
xy = read.csv('normaldata.csv')

q = data.frame(x=quantile(xy$x), y=quantile(xy$y))

caption = 'These quantiles came from a normal distribution'

qtab = xtable::xtable(q, caption=caption, label='tab:quantiles')

print(qtab, file='quantiles.tex')

pdf('histogram.pdf')
hist(xy$x)
dev.off()
